Mathlib/Data/Countable/Basic.lean

```diff
@@ -65,12 +65,19 @@ instance Sum.uncountable_inl [Uncountable α] : Uncountable (α ⊕ β) :=
 instance Sum.uncountable_inr [Uncountable β] : Uncountable (α ⊕ β) :=
   inr_injective.uncountable
 
-instance [Countable α] : Countable (Option α) :=
+instance Option.instCountable [Countable α] : Countable (Option α) :=
   Countable.of_equiv _ (Equiv.optionEquivSumPUnit.{_, 0} α).symm
 
+instance WithTop.instCountable [Countable α] : Countable (WithTop α) := Option.instCountable
```

Comment:
These are new. Do you mind splitting them into a dependent PR? I'll maintainer merge this one...